ostia-operator/test/setup.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+OPENSHIFT_PUBLIC_HOSTNAME=$1
+
+[[ $(oc cluster status) =~ "not running" ]] && oc cluster up --public-hostname="${OPENSHIFT_PUBLIC_HOSTNAME}"
+
+oc login -u system:admin --insecure-skip-tls-verify=true
+oc adm policy add-cluster-role-to-user cluster-admin developer
+oc login -u developer
+oc create -f ../deploy/crd.yaml &> /dev/null || true
+oc adm policy add-cluster-role-to-user cluster-admin system:serviceaccount:ostia:default
```

Comment:
We can move this to the operator deployment test

Reply:
And using the short way `oc adm policy add-cluster-role-to-user cluster-admin -z default` from inside the proper project.